src/components/gltf-model-plus.js

```diff
@@ -181,11 +181,12 @@ function nextTick() {
   });
 }
 
-function cachedLoadGLTF(src, preferredTechnique, onProgress) {
+function cachedLoadGLTF(src, basePath, preferredTechnique, onProgress) {
   // Load the gltf model from the cache if it exists.
   if (!GLTFCache[src]) {
     GLTFCache[src] = new Promise((resolve, reject) => {
       const gltfLoader = new THREE.GLTFLoader();
+      gltfLoader.path = basePath;
```

Comment:
see comment on `basePath` below

Reply:
the intent here is that setting undefined is the same as not setting it. GLTFLoader doesnt require a path to be set.